OptimobileShared/MediaHelper.swift

```diff
@@ -3,17 +3,28 @@
 import Foundation
 
 enum MediaHelper {
-    /// Use ``Region.US`` as fallback region.
-    static let mediaResizerBaseUrl: String = "https://i-us-east-1.app.delivery"
```

Comment:
What do you think, if in case of the absence of the storage will replace a notification content with “Error: Check Optimove AppGroup capabilities” text?
It should help us to reduce the amount of integration issues related to the n—image issues.

Reply:
After some playground, I found that optimobile Core data throwing an no db file.